homeassistant/components/androidtv/media_player.py

```diff
@@ -327,7 +327,9 @@ def __init__(self, aftv, name, apps, turn_on_command,
         self._device_properties = self.aftv.device_properties
         self._is_volume_muted = None
         self._unique_id = 'androidtv-{}-{}'.format(
-            name, self._device_properties['serialno'])
+            name,
+            self._device_properties.get(
+                'serialno', aftv.host.split(':')[0].replace('.', '_')))
```

Comment:
A host can change and so should not be used as unique id. In that case set it to none

Reply:
Fixed. Thanks for the review!